geometry_msgs/Point32[15] balls
geometry_msgs/Point32 white_ball
geometry_msgs/Point32 cue_position
float32[3] cue_orientation
